Templating/Helper/DateTimeHelper.php

```diff
@@ -155,17 +159,21 @@ public function process(\IntlDateFormatter $formatter, \Datetime $date)
     /**
      * Gets a date time instance by a given data and timezone.
      *
-     * @param \Datetime|string|int $data     Value representing date
-     * @param null|string          $timezone Timezone of the date
+     * @param \DateTime|\DateTimeInterface|string|int $data     Value representing date
+     * @param null|string                             $timezone Timezone of the date
      *
-     * @return \Datetime
+     * @return \DateTime
      */
     public function getDatetime($data, $timezone = null)
     {
         if ($data instanceof \DateTime) {
             return $data;
         }
 
+        if ($data instanceof \DateTimeImmutable) {
+            return \DateTime::createFromFormat(\DateTime::ATOM, $data->format(\DateTime::ATOM));
```

Comment:
So if I well understood your comment: https://github.com/sonata-project/SonataIntlBundle/pull/134/files#r67599830

The `\DateTimeImmutable` is transformed onto `\DateTime` just for BC.

No risk of side effect? cc @greg0ire 

So I think it will just be `return $data;` like for `DateTime` objects on next major, right? In this case could you please add a `NEXT_MAJOR` note for this?